proto/dataplane.proto

```diff
@@ -65,6 +66,186 @@ message OspfConfig {
   optional string vrf = 2;
 }
 
+/* ========================== */
+/* ==== Dataplane Status ==== */
+/* ========================== */
+
+enum InterfaceStatusType {
```

Comment:
Should it be OperStatus to make it clear and distinct from any other status, such as admin?